src/main/java/pl/uj/edu/JImageStream/api/ByteListCollector.java

```diff
@@ -0,0 +1,22 @@
+package pl.uj.edu.JImageStream.api;
+
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBufferByte;
+import java.awt.image.WritableRaster;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ByteListCollector implements Collector<List<Byte>> {
+    @Override
+    public List<Byte> collect(BufferedImage bufferedImage) {
+        WritableRaster writableRaster = bufferedImage.getRaster();
+        DataBufferByte dataBufferByte = (DataBufferByte) writableRaster.getDataBuffer();
+        byte[] data = dataBufferByte.getData();
+        List<Byte> list = new ArrayList<>();
```

Comment:
Is it possible to merge lines 13-15 into one? We don't really use DataBufferByte and WritableRaster here